src/Query/Expression.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ClickHouseDB\Query;
+
+/**
+ * Query expression
+ *
+ * @package ClickHouseDB
+ */
+class Expression
+{
+    /**
+     * @var string
+     */
+    protected $expression = '';
```

Comment:
IMO there should not be a default value and not be protected. Let there only be a constructor as the way to create a valid object.
```suggestion
    private $expression;
```

Reply:
Done. Sorry, just now understood that this change could be applied right in github.

Reply:
Np, new feature introduced yesterday ;)